core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -145,6 +149,8 @@ public void start(Boolean isPrimary) throws IOException {
         }
       }
       mAsyncJournalWriter = new AsyncJournalWriter(mJournalWriter);
+      mAsyncAuditLogWriter = new AsyncUserAccessAuditLogWriter();
+      mAsyncAuditLogWriter.start();
```

Comment:
This will create one audit logger per master, but we only want a single audit logger for the filesystem master